src/components/Input.react.js

```diff
@@ -62,7 +62,7 @@ export default class Input extends Component {
                     if (setProps) {
                         const payload = {
                             n_blur: this.props.n_blur + 1,
-                            n_blur_timestamp: new Date(),
+                            n_blur_timestamp: new Date.now(),
```

Comment:
This is a breaking change https://github.com/plotly/dash-core-components/issues/467 - yes, we should do it, but we should call it out.